webextensions/api/devtools.json

```diff
@@ -51,16 +51,100 @@
               }
             }
           },
+          "Resource": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/inspectedWindow/resource",
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
```

Comment:
Does "mirror" automatically indicate that this only came in after Edge adopted Chromium?

Reply:
@dotproto That is my understanding

Reply:
I can confirm this.